src/components/Link/__tests__/Link.test.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import {render, screen} from '@testing-library/react';
+import {Link} from '../Link';
+
+describe('Link', () => {
+    test('Link is anchor', () => {
+        render(<Link href="#" />);
+
+        const link = screen.getByRole('link');
+
+        expect(link).toBeInTheDocument();
+    });
+
+    test('Link is span', () => {
+        render(<Link qa="test" />);
+
+        // btw there should be role attribute at span
```

Comment:
What role should it be?

Reply:
"link"
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/link_role